src/modules/api/api.js

```diff
@@ -0,0 +1,52 @@
+import axios from 'axios';
+
+const baseUrl = process.env.BASE_API_URL;
+const createGames = async () => {
+  try {
+    const res = await axios.post(`${baseUrl}games/`, {
+      name: 'My cool new game',
+    });
+    return (res.data.result);
+  } catch (error) {
+    return error;
+  }
+};
+
+class API {
+  constructor() {
+    this.id = process.env.GAME_ID || '';
+    this.url = baseUrl || '';
+  }
+
+  addApi = async ({ user, score }) => {
+    try {
+      const res = await axios.post(`${this.url}games/${this.id}/scores`, {
+        user,
+        score,
+      });
+      if (res.status === 201) {
+        return (res.data.result);
+      }
+      throw new Error('Adding failed');
+    } catch (error) {
+      throw new Error('Adding failed');
+    }
+  }
```

Comment:
- [ ] Just as the previous reviewer noted, both the get and the post are not hitting the API. It returns 'successfully failed' whereas, it should post and fetch the data. Kindly take a second look at it.

Reply:
@Olamarx Have you added `.env` file? How come it is working here?